### PR TITLE
perf: optimize lighthouse score

### LINKED ISSUE #216: performance: Get a 100% score on LightHouse metrics `/playground`
Ensure the release build of the website gets a 100% score on `/playground`

NOTE: Ignore PWA

Screenshot 2024-04-25 at 9 11 24 AM

### PR DESCRIPTION
**Pull request by neo773**
Optimizations:
- Patched GraphiQL to load fonts via CDN, reducing CSS Bundle size from 525kb to 165kb (-68%  reduction)
- Optimized Lottie animations files reducing payload by another 50%
- Optimized all PNGs and JPEGs
- Some accessibility issues

For accessibility I've fixed the 1st party issues rest all are caused by 3rd party like GraphiQL

Report for /playground
image

Report for /
image

/claim #216
/claim #217

fixes #216
fixes #217

### CODE REVIEW

File: static/install.sh [hunk 1]
```diff
@@ -1,69 +0,0 @@
-#!/bin/bash
```
Comment (amitksingh1490) [inline static/install.sh:1]: why is this deleted?

**Comment by neo773** [comment]
@amitksingh1490 
Fixed!

**Comment by amitksingh1490** [comment]
@neo773 can you deploy this change on https://neo773.github.io/tailcallhq.github.io/. want to check the lighthouse score now

**Comment by amitksingh1490** [comment]
@neo773 Screenshot @tusharmath shared in the issue was for mobile audit and you have shared it for desktop audit.
Last deployed changes on https://neo773.github.io/tailcallhq.github.io/playground/
is not close to 100% on both the pages
image
image

**Comment by raikasdev** [comment]
Google's pagespeed is a more realistic Lighthouse run being not affected by eg. extensions: https://pagespeed.web.dev/analysis/https-neo773-github-io-tailcallhq-github-io/50w4z55rwq?form_factor=mobile

image

**Comment by neo773** [comment]
@raikasdev 

Sorry, I have to disagree on this one. 
Lighthouse is actually more accurate here, as PageSpeed is flagging issues that are out of my control, such as "Reduce JavaScript execution time." This includes the Lottie animation library, which alone adds 320.0 kB to the bundle.

Lighthouse's diagnostic is also inaccurate as it's doing CSS coverage for home page and excluding Docusaurus content.
image

image

**Comment by neo773** [comment]
Also I pushed a change that adds avif images

**Comment by raikasdev** [comment]
Pagespeed is Lighthouse, but it just runs on Google's servers. That's the environment the search engine cares for, if the goal here is to get better SEO and load times.

> Lighthouse's diagnostic is also inaccurate as it's doing CSS coverage for home page and excluding Docusaurus content.

It's not inaccurate. It means that there is useless CSS rules loaded on the home page that are only on Docusaurus content pages.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
           npm run build
 
       - name: Deploy 🚀
-        if: github.ref == 'refs/heads/develop'
+        # if: github.ref == 'refs/heads/develop'
```
Comment (tusharmath) [inline .github/workflows/ci.yml:35]: revert the CI changes.
